Solutions/IPinfo/Data Connectors/Company/IPinfo_Company_API_AzureFunctionApp.json

```diff
@@ -0,0 +1,114 @@
+{
+    "id": "IPinfoCompanyDataConnector",
+    "title": "IPinfo",
```

Comment:
Please keep title of the connectors **unique**

All the connectors shares same title